torch/csrc/autograd/FunctionsManual.h

```diff
@@ -230,6 +230,14 @@ std::tuple<Tensor, Tensor> householder_product_backward(const Tensor& grad, cons
 std::tuple<Tensor, Tensor> polar_backward(
     const Tensor& grad,
     const Tensor& result);
+Tensor i1_backward(
+    const Tensor& grad,
+    const Tensor& self,
+    const Tensor& result);
+Tensor i1e_backward(
+    const Tensor& grad,
+    const Tensor& self,
+    const Tensor& result);
 Tensor lu_unpack_backward(
   const std::vector<torch::autograd::Variable>& grads,
```

Comment:
Is it ok to make this `const vectorlist& grads` in this PR or leave it as is?

https://github.com/pytorch/pytorch/blob/c371542efc31b1abfe6f388042aa3ab0cef935f2/torch/csrc/autograd/function_hook.h#L12